deepcell_label/blueprints.py

```diff
@@ -33,6 +34,33 @@ def health():
     return jsonify({'message': 'success'}), 200
 
 
+def get_git_commit() -> str:
+    return (
+        subprocess
+        .check_output(['git', 'show', '-s', '--oneline', 'HEAD'])
+        .decode('ascii')
+        .strip()
+    )
+
+
+def get_git_branch() -> str:
+    return (
+        subprocess
+        .check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD'])
+        .decode('ascii')
+        .strip()
+    )
+
```

Comment:
What does this do if `git` is not installed or the `.git` folder can't be accessed?

Is there a way we can set the commit/branch in `config.py` and return them here?